docs/_articles/en/deploy-changes/metadata-operations.md

```diff
@@ -2,24 +2,14 @@
-
-
-## Source Diff
 
-The Source Diff command makes it easier to visualize the changes between files in your local project and the metadata in your org. You can look at diffs for an individual file or directory.
-Right-click on a file or folder and run **SFDX: Diff File Against Org** or **SFDX: Diff Folder Against Org**.  
-
-![Source Diff command](../../../images/source_diff.png)
+## Overview
 
-View individual file diffs side by side. A file doesn't show up in the list of diffs if it doesn’t exist locally or in the org.
+Say you want to change an existing Apex class or add a custom object to your production org. You first make and test these changes in a scratch org, check for conflicts, and then deploy the changes to another org for your team to access them. This ability to move changes between different Salesforce development environments such as scratch orgs and sandboxes, for various stages of testing before launching into production, is key to Salesforce development. Use these metadata operations to deploy your local changes to an org as a part of your ALM process. With conflict detection turned on, you don't have to worry about accidentally overwriting others' changes in the org. You can track changes by easily viewing changes between your local files and your default org at any time. If you’re working in a team, you can retrieve the latest changes in your org on to your local Salesforce project.
 
 ## Retrieve Source
 
-Use the **SFDX: Retrieve Source from Org** command to bring metadata from the default org into your local project.  For non-source-tracked orgs, VS Code doesn’t automatically track changes to your org, so be sure to track the changes you retrieve. Retrieving source from an org overwrites the local versions of the source files.
+Use the **SFDX: Retrieve Source from Org** command to bring metadata from the default org into your local project. For non-source-tracked orgs, VS Code doesn’t automatically track changes to your org, so be sure to track the changes you retrieve. Retrieving source from an org overwrites the local versions of the source files.
```

Comment:
(re: non-source-tracked orgs) So you'll have to manually keep track of any changes you retrieve. Just to clarify the HOW part. that it's a manual process.

Reply:
For source tracked changes, the CLI used to display a status that indicated that the remote and local files were different - we don't do that anymore? We just overwrite regardless?? I should test this.